contracts/contracts/harvest/OETHHarvesterSimple.sol

```diff
@@ -0,0 +1,107 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import { Governable } from "../governance/Governable.sol";
+import { IStrategy } from "../interfaces/IStrategy.sol";
+import { IERC20 } from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+
+contract OETHHarvesterSimple is Governable {
+    ////////////////////////////////////////////////////
+    /// --- STORAGE
+    ////////////////////////////////////////////////////
+    address public operator;
+    address public strategist;
+    mapping(address => bool) public isAuthorized;
+
+    ////////////////////////////////////////////////////
+    /// --- EVENTS
+    ////////////////////////////////////////////////////
+    event Harvested(address token, uint256 amount);
+    event OperatorSet(address operator);
+    event StrategistSet(address strategist);
+    event StrategyStatusSet(address strategy, bool status);
+
+    ////////////////////////////////////////////////////
+    /// --- MODIFIERS
+    ////////////////////////////////////////////////////
+    modifier onlyOperator() {
+        require(
+            msg.sender == operator || isGovernor(),
+            "Only Operator or Governor"
+        );
+        _;
+    }
+
+    ////////////////////////////////////////////////////
+    /// --- CONSTRUCTOR
+    ////////////////////////////////////////////////////
+    constructor(
+        address _governor,
+        address _operator,
+        address _strategist
+    ) {
+        require(_strategist != address(0), "Invalid strategist");
+        operator = _operator;
+        strategist = _strategist;
+        _setGovernor(_governor);
+    }
+
+    ////////////////////////////////////////////////////
+    /// --- MUTATIVE FUNCTIONS
+    ////////////////////////////////////////////////////
+    function harvestAndTransfer(address _strategy) external onlyOperator {
+        _harvestAndTransfer(_strategy);
+    }
+
+    function harvestAndTransfer(address[] calldata _strategies)
+        external
+        onlyOperator
+    {
+        for (uint256 i = 0; i < _strategies.length; i++) {
+            _harvestAndTransfer(_strategies[i]);
+        }
+    }
+
+    function _harvestAndTransfer(address _strategy) internal {
+        // Ensure strategy is authorized
+        require(isAuthorized[_strategy], "Strategy not authorized");
```

Comment:
you can revert on 0 address _strategy here as well

Reply:
Agree! Changed on this commit: [b79578b](https://github.com/OriginProtocol/origin-dollar/pull/2333/commits/b79578b4f0b02d6fcfb99beede7c18b761086c24)

Reply:
I think fine to not have an extra require here. The isauthorized covers it, and the later function call would fail regardless.